package.yaml

```diff
@@ -83,6 +83,7 @@ executables:
     ghc-options:
     - -rtsopts
     - -O2
+    - -threaded
```

Comment:
As we discussed yesterday, can this go away for now and it still work? That way we can delay making this actually execute multithreaded but keep this architecture until we start using SMT-Lib?

Reply:
Tested and it worked!